src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Logging/DefaultLogHubLogWriter.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Logging
+{
+    internal class DefaultLogHubLogWriter : LogHubLogWriter, IDisposable
+    {
+        private TraceSource _traceSource = null;
+
+        public DefaultLogHubLogWriter(TraceSource traceSource)
+        {
+            _traceSource = traceSource;
```

Comment:
Does a null `TraceSource` ever make sense, or might this class be eligible for `#nullable enable`.

Reply:
Good point, updated.